libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -438,6 +438,7 @@ dictionary ReverseSwapFeesRequest {
 dictionary OnchainPaymentLimitsResponse {
     u64 min_sat;
     u64 max_sat;
+    u64 max_sat_with_current_channels;
```

Comment:
I think  it is better to have one field max_sat and set there the minimum of max amount based on the users balance and the value returned from the swapper.

Reply:
Good idea, done: ee8a9ef0856b7d0d98af4891944e3dc532929510

I also covered the case when `max_reverse_swap_amount` would return a value below the Boltz `min`. With the naive implementation it would lead to

```
min_sat: 50_000, // From Boltz
max_sat: 0       // min(Boltz max, max_reverse_swap_amount)
```

Both `min_sat` and `max_sat` are now set to 0 in that case, to indicate no reverse swap is possible.